java-bootcamp/src/main/java/com/globant/service/UserService.java

```diff
@@ -0,0 +1,52 @@
+package com.globant.service;
+
+/**
+ * "UserService" defines behavior a user service, implements to "IUserService".
+ * Implements basics operations such as Create, Read, Update and Delete user.
+ * @author Herminio Andres Hernandez
+ * @version 21/07/2018
+ */
+
+public class UserService implements IUserService{
+	//Attributes of the class.
+	private UserRepository userRepository;
+	/**
+	 * Constructor of user service.
+	 */
+	public UserService() {
+		userRepository = new UserRepository();
+	}//Close constructor
+	/**
+	 * The method allows to create user. 
```

Comment:
You don't need to replicate the javadoc in the implementation, if you add this to the interface it's enough